### ISSUE TITLE
Generate and link text-only schedule

### ISSUE THREAD

**Issue by jmatth** [jmatth/dextools#52]
Double Exposure currently provides text-only versions of the schedule on request via email. It should be easy enough generate our own version either in the scraper or dynamically on the website and add it to the site navigation.
